src/common/domains/printable.ml

```diff
@@ -176,6 +176,7 @@ struct
 
   let lift m = {m; lazy_hash = LazyHash.make m}
   let unlift {m; _} = m
+  let relift x = x
```

Comment:
Would I not need to call `relift` for `M` and reset cached hashes here?

Reply:
Why should the hashes change?

Reply:
For future reference: This is now included in the next commit to be sure.